apps/frontend/src/components/Hero.tsx

```
@@ -119,41 +127,41 @@ const Hero = () => {
             >
               <motion.div
                 whileHover={{ scale: 1.02, transition: { duration: 0.2 } }}
-                className="relative bg-online rounded-3xl p-6 overflow-hidden cursor-pointer shadow-lg group"
+                className="relative bg-online rounded-2xl md:rounded-3xl p-4 md:p-6 overflow-hidden cursor-pointer shadow-lg group"
                 role="button"
                 tabIndex={0}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)